angular.json

```diff
@@ -62,7 +62,9 @@
                 "lazy": true
               }
             ],
-            "scripts": []
+            "scripts": [
```

Comment:
This is not required as you included the script via a cdn in `index.html` (which is a better way). Remove this change (file) from your commit.